gradio/utils.py

```diff
@@ -225,13 +224,16 @@ def same_children_recursive(children1, chidren2, mapping):
     same_children_recursive(children1, children2, mapping)
 
     for d1, d2 in zip(config1["dependencies"], config2["dependencies"]):
-        for t1, t2 in zip(d1["targets"], d2["targets"]):
+        for t1, t2 in zip(d1.pop("targets"), d2.pop("targets")):
             assert mapping[t1] == t2, "{} does not match {}".format(d1, d2)
-        assert d1["trigger"] == d2["trigger"], "{} does not match {}".format(d1, d2)
-        for i1, i2 in zip(d1["inputs"], d2["inputs"]):
+        for i1, i2 in zip(d1.pop("inputs"), d2.pop("inputs")):
             assert mapping[i1] == i2, "{} does not match {}".format(d1, d2)
-        for o1, o2 in zip(d1["outputs"], d2["outputs"]):
+        for o1, o2 in zip(d1.pop("outputs"), d2.pop("outputs")):
             assert mapping[o1] == o2, "{} does not match {}".format(d1, d2)
+        # status tracker is ignored
```

Comment:
Why do we ignore status tracker?

Reply:
The reason we ignore the status tracker is because it is going to have different IDs between different blocks, just like input and output components. In the case of inputs and outputs, we keep a mapping dictionary to make sure all of the dependencies in the blocks are equivalent (the same input/output combinations are used in each event). However, status tracker ids are never used for anything else, so we can just delete them. I'll add a comment to explain in the code